### PR TITLE
Update links with GH public repo URL

### PR DESCRIPTION
**Pull request by ibarrae**

### CODE REVIEW

File: src/components/ui/GoToRepoButton/GoToRepoButton.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import GitHubIcon from '@assets/svg/github-logo.svg'
```
Comment (JoseRFelix) [inline src/components/ui/GoToRepoButton/GoToRepoButton.tsx:1]: This file should go in the `components/common` since it's a derivative of a UI component.

**Review by JoseRFelix — CHANGES_REQUESTED** [review]
Looks good! Left one comment and from there we can merge.

File: src/components/ui/GoToRepoButton/GoToRepoButton.tsx (hunk 1 above)
Comment (ibarrae) [reply src/components/ui/GoToRepoButton/GoToRepoButton.tsx:1]: Thanks for the catch! Working on it :+1:

**Review by JoseRFelix — APPROVED** [review]
Thanks for the update; LGTM :rocket:

**Pull request merged**